src/Swarm/Language/Syntax.hs

```diff
@@ -100,14 +102,27 @@ import Swarm.Language.Types
 import Swarm.Util qualified as Util
 import Witch.From (from)
 
+-- | Maximum perception distance for
+-- 'chirp' and 'sniff' commands
+maxSniffRange :: Int32
+maxSniffRange = 200
+
 ------------------------------------------------------------
 -- Directions
 ------------------------------------------------------------
 
 -- | An absolute direction is one which is defined with respect to an
 --   external frame of reference; robots need a compass in order to
 --   use them.
-data AbsoluteDir = DNorth | DSouth | DEast | DWest
+--
+-- NOTE: These values are ordered by increasing angle according to
+-- the standard mathematical convention.
+-- That is, the right-pointing direction, East, is considered
+-- the "reference angle" and the order proceeds counter-clockwise.
+-- See https://en.wikipedia.org/wiki/Polar_coordinate_system#Conventions
+--
+-- Do not alter this ordering, as there exist functions that depend on it.
```

Comment:
This makes sense.  Just for completeness' sake, it might be nice to include a reference to an example of a function which depends on the ordering.